pcl_ros/CMakeLists.txt

```diff
@@ -118,8 +119,20 @@ ament_target_dependencies(pcl_ros_tf
 #
 ### Tools
 #
-#add_executable(pcd_to_pointcloud tools/pcd_to_pointcloud.cpp)
-#target_link_libraries(pcd_to_pointcloud ${Boost_LIBRARIES} ${catkin_LIBRARIES} ${PCL_LIBRARIES})
+add_library(pcd_to_pointcloud_lib SHARED tools/pcd_to_pointcloud.cpp)
+# add_executable(pcd_to_pointcloud tools/pcd_to_pointcloud.cpp)
```

Comment:
Does this need to be here? I don't see a `main` anymore for that to be possible. 

Though it might be nice to add a new main in a new file so that there is a standalone version available